data/112/580/545/1/1125805451.geojson

```diff
@@ -18,6 +18,7 @@
     "lbl:min_zoom":13.5,
     "mz:hierarchy_label":1,
     "mz:is_current":-1,
```

Comment:
We're marking it funky, but is it current or not?

Reply:
This is a locality point that falls within a newly-updated locality polygon. I'll add a commit to flag records with `mz:is_funky` of 1 to also get a `mz:is_current` of 1.

Reply:
Taken care of in 21c8566